src/wallet/wallet.h

```diff
@@ -696,7 +696,18 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     /* HD derive new child key (on internal or external chain) */
     void DeriveNewChildKey(CKeyMetadata& metadata, CKey& secret, bool internal = false);
 
-    std::set<int64_t> setKeyPool;
+    bool fFileBacked;
```

Comment:
Oh. That's a rebase issue. `fFileBackand` was removed in the meantime, Will remove it here.